examples/07-Train-an-xgboost-model-using-the-Merlin-Models-API.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>nit: My sugg is to use present tense nearly all the time: "The <code>get_movielens</code> function downloads the <code>movielens-100k</code> data for us and returns it materialized as a Merlin <code>Dataset</code> ."</p><p> </p><p>Present tense is typically easier for the reader to understand and lends confidence and authority to your voice. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/522/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>